regexlint/checkers.py

```diff
@@ -413,42 +406,12 @@ def check_multiline_anchors(reg, errs):
         errs.append((num, level, anchor.start, msg))
 
 
-def check_wide_unicode(reg, errs):
-    num = '121'
-    level = logging.WARNING
-    msg = 'Wide unicode causes problems in narrow builds'
```

Comment:
This code tested that patterns in Pygments that needed to use [`unirange`](https://github.com/pygments/pygments/blob/master/pygments/util.py#L219) in Pygments actually do.  It looks like this isn't necessary anymore after PEP 393, so the `unirange` in `pygments/util.py` as well as patching `sys.maxunicode` in `regexlint/parser.py` can also go away.

Reply:
Yep, that's already gone.